stable/ghost/templates/deployment.yaml

```diff
@@ -128,6 +128,10 @@ spec:
         - name: SMTP_SERVICE
           value: {{ .Values.smtpService | quote }}
         {{- end }}
+        {{- range $key, $value := .Values.env }}
+        - name: {{ $key }}
+          value: {{ $value | quote }}
+        {{- end }}
```

Comment:
use `toYaml` instead 

```
            {{- if .Values.extraEnvVars }}
            {{- toYaml .Values.extraEnvVars | nindent 8 }}
            {{- end }}
```

Reply:
This suggestion wouldn't work due to the way deployment.yaml `env:` is an array of hashes while `extraEnvVars` is just a hash; that's why I had to use range. Using `toYaml` would means a lot of typing in the overrides file.

Reply:
@trankchung you are correct about having to type a more, however the `toYaml` method also allows users to specify  environment variables that consumes secrets which is why I prefer that method

Reply:
@sameersbn What you said made sense so I've changed it to how you suggested.